app/src/main/java/de/danoeh/antennapod/fragment/InboxFragment.java

```diff
@@ -33,11 +36,16 @@ public class InboxFragment extends EpisodesListFragment {
     private static final String PREF_DO_NOT_PROMPT_REMOVE_ALL_FROM_INBOX = "prefDoNotPromptRemovalAllFromInbox";
     private SharedPreferences prefs;
 
+    private DateSortType sortType = UserPreferences.getInboxSortOrder();
+
     @NonNull
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         final View root = super.onCreateView(inflater, container, savedInstanceState);
         toolbar.inflateMenu(R.menu.inbox);
+
+        SortMenuBuilder.addSortMenu(toolbar.getMenu());
```

Comment:
I think you can simply call `toolbar.inflateMenu()` again with another menu xml file to add additional menu items. That way, you don't need to specify custom constants for the IDs in `SortMenuBuilder`, which has the risk of creating duplicate IDs in the future.